lessons/creating-apis-with-python-and-flask.md

```diff
@@ -0,0 +1,713 @@
+
+{% include figure.html filename="hugo.png" caption="Scatterplot of first sentence length against date of publication." %}
+
+As new Hugo winners were added to the database, the script that generated this visualization would immediately be able to use the new information. If the visualization were created in D3 or another web-based utility, this plot would actually reflect additional data added to the book archive as soon as the archive was updated—that is, in real time. As additional data accrued, we might, for example, learn if John Scalzi's unusually lengthy opening to his 2013 *Red Shirts* was an aberration or the continuation of a longer trend toward wordiness in science fiction. Conversely, if your API were to change its URL structure or cease to function, applications based on it will no longer work. Remember that, when creating an API, you are assuming some responsibility for the applications that others may build with it.
+
+A strong API can be considered the backbone of a potentially limitless number of projects or avenues of research. Though the above example takes the form of a visualization of the limited amount of data we've provided in our Distant Reading Archive, a project based on this API might just as easily take the form of a Twitterbot that shares first sentences or a library webpage that displays book openings and year of publication alongside other book metadata. In many cases, it makes sense to first create an API interface to your core data or functionality before extrapolating on it to create a visualization, application, or website. Not only does it make your work accessible to researchers working on other projects, but it often leads to a more comprehensible and maintainable project.
+
+# Resources
+
+The below resources provide information on useful APIs for researchers in the humanities and social sciences as well as further reading on API concepts.
+
+## APIs for Humanities Researchers
+
+[Chronicling America \(Library Of Congress\)](http://chroniclingamerica.loc.gov/) - A digitized collection of American newspaper articles from the 18th to the 20th century.
+
+[Connecting Repositories \(CORE\)](https://core.ac.uk/) - A collection of open access articles from various sources hosted by the Open University.
+
+[English Broadside Ballad Archive \(EBBA\)](https://diggingintodata.org/repositories/english-broadside-ballad-archive-ebba)
+
+[History Data Service (HDS)](http://hds.essex.ac.uk/) - A collection of data from a wide variety of historical sources.
+
+[ARTstor](http://www.artstor.org/) - More than two million images from historical, carographic, and art history sources.
```

Comment:
I was surprised to see Artstor on here because I wasn't aware they had an API for accessing their library... and I actually couldn't find one on their site after paging around a bit. Is there a more specific link to an API for this? Or should it be replaced by another API service like Europeana?